CorsixTH/Lua/cheats.lua

```diff
@@ -50,21 +50,89 @@ function Cheats:Cheats(hospital)
     {name = "increase_prices", func = self.cheatIncreasePrices},
     {name = "decrease_prices", func = self.cheatDecreasePrices},
   }
+
+  --[[ Toggle-based cheats are found in below function ]]
+end
+
+--! A private function to return the toggle_cheats list (code operated cheats)
+--! These cheats do not appear in the Cheat UI menu, and are only accessed from fax code inputs currently.
+--! They contain adviser strings, so cannot be persisted.
+--!return the cheat list
+function Cheats._toggleCheatList()
+  local toggle_cheats = {
```

Comment:
Why is this table wrapped in a function?

Reply:
It might not need to be, but it can't exist in self/permanent object. Strings can not be persisted (or should not, really).
It could just sit outside the Cheats:Cheats function by itself.

Reply:
Please do, there are performance implications in lua (less so lua_jit)

Reply:
This has actually caused a crash making that change, so will need to look into it next week

```

Reply:
Yep, it's been set up as such already. Just got to ponder why it doesn't like it

Reply:
Found the problem, silly lua